### ISSUE TITLE
'input' Event doesn't fire when change value programmatically

### ISSUE THREAD

**Issue by alexhyriavets** [shentao/vue-multiselect#1187]
### Reproduction Link
https://jsfiddle.net/AlexanderHyriavets/caxfLs4u/5/

### Steps to reproduce
1. Click 'change value' button
2. See console

### Expected behavior
Emits 'input' when the value changes

### Actual behavior
Does not emit 'input' event when the value changes

Description of the 'input' event in docs says: *Emitted after this.value changes*. But in this particular case it is not the truth.

**Comment by dwieringa** [comment]
Maybe the resolution should be to change the documentation to state "Emitted after this.value is changed by user action."  I am currently relying on the fact that this event only fires on user actions.

**Comment by dwieringa** [comment]
https://github.com/shentao/vue-multiselect/issues/1191 is related.  When using `:taggable="true"`, the `addTag` event handler you provide is responsible for updating the value.

See https://vue-multiselect.js.org/#sub-tagging

```
The event handler should add the received tag to both the options list and the value.
```

**Comment by JeroenKocyigit** [comment]
@dwieringa & @alexhyriavets 
Have you found a workaround for this by any chance?

**Comment by AllanOricil** [comment]
for some reason now the jsfiddle works. How is that possible? I have the same version in my app and it does not work there. I read the entire source code and there is one emit('input') when multiselect detects the value has changed. But it emits [] or null 
That is really weird, by the way...

**Comment by AllanOricil** [comment]
https://github.com/shentao/vue-multiselect/issues/1392

**Comment by mariusa** [comment]
For me , `@input` never triggers. `@select / @remove` work.

**Comment by TeaDrinkingProgrammer** [comment]
For people wondering why @input doesn't work on next/Vue 3: this is because of a [Vue 3 change]( https://v3-migration.vuejs.org/breaking-changes/v-model.html)) that is not reflected in the vue-mutliselect docs.
TLDR: change @input to @update:modelValue or @update:model-value to get it to work again
